pkgs/development/web/bun/default.nix

```
@@ -1,40 +1,10 @@
-{ stdenvNoCC, callPackage, fetchurl, autoPatchelfHook, unzip, openssl, lib }:
-let
-  dists = {
-    aarch64-darwin = {
-      arch = "aarch64";
-      shortName = "darwin";
-      sha256 = "sha256-R17Ga4C6PSxfL1bz6IBjx0dYFmX93i0y8uqxG1eZKd4=";
-    };
-
-    aarch64-linux = {
-      arch = "aarch64";
-      shortName = "linux";
-      sha256 = "sha256-KSC4gdsJZJoPjMEs+VigVuqlUDhg4sL054WRlAbB+eA=";
-    };
+{ stdenvNoCC, callPackage, fetchurl, autoPatchelfHook, unzip, openssl, lib, writeShellScript, curl, jq, common-updater-scripts }:
```

Too long.
Please put one attribute per line:

```suggestion
{ lib
, stdenvNoCC
, callPackage
, fetchurl
, autoPatchelfHook
, unzip
, openssl
, writeShellScript
, curl
, jq
, common-updater-scripts
}:
```